packages/synapse-react-client/src/lib/containers/download_list/DownloadConfirmation.tsx

```diff
@@ -299,6 +299,8 @@ export const DownloadConfirmation: React.FunctionComponent<
   return (
     <>
       <Alert
+        sx={{ pr: 4, py: 1 }}
+        icon={StatusConstruct[status].severity === 'error' ? undefined : false}
```

Comment:
Only show the Alert icon when logged out -- the other Alert messages start with icons, either the spinner while loading/processing or the DownloadDetails file icon